plugins/source/aws/resources/services/iam/users.go

```diff
@@ -137,7 +137,7 @@ func IamUsers() *schema.Table {
 				Name:        "tags",
 				Description: "A list of tags that are associated with the user",
 				Type:        schema.TypeJSON,
-				Resolver:    schema.PathResolver("User.Tags"),
+				Resolver:    client.TagsToMap("User.Tags"),
```

Comment:
I think it should be this:
```suggestion
				Resolver:    client.ResolveTags,
```